src/content/docs/new-relic-only/drupal-configuration/page-components/manage-content-footers.mdx

```diff
@@ -9,7 +9,7 @@ redirects:
 ---
 
 <Callout variant="important">
-  Only the Docs team can edit and create new footer blocks. If you'd like to request additions, changes, or deletions to footer content, contact `@hero` in the [#documentation](https://newrelic.slack.com/messages/documentation) Slack channel or send an email to [docs@newrelic.com](mailto:docs@newrelic.com).
+  Only the Docs team can edit and create new footer blocks. If you'd like to request additions, changes, or deletions to footer content, contact `@hero` in the [#documentation](https://newrelic.slack.com/messages/documentation) Slack channel (internal New Relic) or [create a GitHub issue](https://github.com/newrelic/docs-website/issues/new?assignees=&labels=enhancement%2C+content&template=content-enhancement.md&title=Request+some+docs+improvements%21).
```

Comment:
Is there a reason why the issue url is longer in this line than the others?

Reply:
I grabbed it from the Docs improvements content issue page. Think I should make it match the other ones, @bradleycamacho?

Reply:
Oh, it's sort of irrelevant here, because that doc is just going to be wiped out.

Reply:
I updated it to match the others.